terracumber_config/tf_files/SUSEManager-4.1-qam.tf

```diff
@@ -1110,6 +1110,7 @@ module "controller" {
   git_repo     = var.CUCUMBER_GITREPO
   branch       = var.CUCUMBER_BRANCH
 
+  grains = { build_validation = "true" }
```

Comment:
I think we are over-engineering here. I would really avoid any extra parameter in the main.tf or sumaform.
We can just inject this variable from the pipeline, using terracumber.

Reply:
No, no, no :-) . Please.

I want to be able to deploy myself (no jenkins) => sumaform.

And even when deployed by Jenkins, when I test manually, I do not want to need to do export BUILD_VALIDATION=true by hand every time i test.